### ISSUE TITLE
Internet access 

### ISSUE THREAD

**Issue by wolf000x** [elementary/switchboard-plug-network#150]
The computer goes into sleep mode after I turn it on the internet network does not work I need to turn it on and off so it works Internet Access

**Comment by peteruithoven** [comment]
Could you please clarify?

Do you mean that after your computer wakes up from sleep and you turn on the internet network it doesn't work. You need to turn it off and on to get it to work?

Also how to you turn on the internet network? Do you enable the Wireless or Wired connection?

**Comment by wolf000x** [comment]
Yes after he wakes up I go to a browser and I search but I do not have internet access I need to turn the connection on and off.

Wireless network.
